kedro_airflow/plugin.py

```diff
@@ -59,51 +49,39 @@ def airflow_commands():
     pass
 
 
-def _get_dag_filename():
-    if KEDRO_VERSION.match(">=0.15.0"):
-        context = get_project_context("context")
-        project_path = context.project_path
-        project_name = context.project_name
-    else:
-        project_path = get_project_context("project_path")
-        project_name = get_project_context("project_name")
-    dest_dir = project_path / "airflow_dags"
-    return dest_dir / (slugify(project_name, separator="_") + "_dag.py")
-
-
 @airflow_commands.command()
-def create():
+@click.option("--pipeline", default="__default__")
+@click.option("--env", default="local")
+def create(pipeline, env):
```

Comment:
```suggestion
@click.option("-t", "--target-dir", "target_path",
              type=click.Path(writable=True, resolve_path=True),
              default="./airflow_dags/")
@click.pass_obj
def create(metadata: ProjectMetadata, pipeline_name, env):
```
Would something like this work? Could we enable users to specify where they want to safe the file and how it is named, if the target is not a directory?